Game/Assets/FoF/Scripts/Player.cs

```diff
@@ -0,0 +1,103 @@
+﻿using UnityEngine;
+using System.Collections;
+using FoF;
+
+namespace FoF
```

Comment:
We'd best update this to Coliseo now, rather than later down the line. Just to unify everything.